MQTT_PORT: 1888
SERVER_IP: "vivernd.com"
CLIENT_ID: "#"